lib/helpers.py

```diff
@@ -11,3 +12,26 @@ def is_ajax(request):
     Request object method is_ajax() was removed in Django 4.0, this can be used instead.
     """
     return request.headers.get('x-requested-with') == 'XMLHttpRequest'
+
+
+def str_in_selected_language(string):
+    """If a string includes alternate translations wrapped with |-characters,
+    return a version of the string with only the text in the language the page
+    is in.
+    If the language alternatives don't include the selected language, return
+    the first alternative.
+    """
```

Comment:
Same thing about the docstring quotes.

Reply:
As discussed above, according to style guides, the first `"""` of a docstring should be on the same row as the string, so I did not make this change.